conf.d/main

```diff
@@ -25,8 +25,9 @@ MYSQL_BATCH="mysql --user=root --password=$MYSQL_PASS --batch"
 MYSQL_ADMIN="mysqladmin --user=root --password=$MYSQL_PASS"
 
 # start services
-service apache2 start
 service mysql start
+service apache2 start
+apt-get -y install php-bcmath
```

Comment:
Unless a packages is being installed from a third party repo (that is configured by overlay and/or conf) then it's much better to put additional packages in the plan ([`plan/main`](https://github.com/turnkeylinux-apps/gnusocial/blob/master/plan/main)) rather than manually installing via apt.

Reply:
Went through and fixed this for you